cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -229,6 +250,36 @@ func (o *Options) Run(args []string) error {
 	}
 	fmt.Fprintf(o.Out, "CertificateRequest %s has been created in namespace %s\n", req.Name, req.Namespace)
 
+	if o.FetchCert {
+		if !apiutil.CertificateRequestHasCondition(req, cmapiv1alpha2.CertificateRequestCondition{
+			Type:   cmapiv1alpha2.CertificateRequestConditionReady,
+			Status: cmmeta.ConditionTrue,
+		}) {
+			fmt.Fprintf(o.Out, "CertificateRequest %v in namespace %v has not been signed yet. Wait until it is signed...\n",
+				req.Name, req.Namespace)
+			timeout := time.After(5 * time.Minute)
```

Comment:
Does this mean the command will give up after 5 minutes?
If so could be nice to have this as a flag... If you look at ACME certs it may take a while depending on how quickly the challenge propagates (read DNS)